bin/preview-plugin.sh

```
@@ -0,0 +1,258 @@
+# Ensure the plugin is in the core list
+if ! echo ",$CORE_NAMES," | grep -q ",$CAMEL_NAME,"; then
+  CORE_NAMES="$CORE_NAMES,$CAMEL_NAME"
+fi
+
+case "$ACTION" in
+  testers)
+    # Add to hidden list if not already there
+    if ! echo ",$HIDDEN_NAMES," | grep -q ",$CAMEL_NAME,"; then
+      HIDDEN_NAMES="$HIDDEN_NAMES,$CAMEL_NAME"
+    fi
+    IFS=',' read -ra TESTER_ARR <<< "$TESTERS"
```

Shellcheck (0.11.0)

[warning] TESTER_ARR appears unused. Verify use (or export if used externally).

(SC2034)